src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1318,61 +1318,93 @@ public override BoundNode VisitConditionalOperator(BoundConditionalOperator node
             VisitCondition(node.Condition);
             var consequenceState = this.StateWhenTrue;
             var alternativeState = this.StateWhenFalse;
+
+            var consequence = VisitConditionalOperand(consequenceState, node.Consequence, isByRef);
+            Unsplit();
+            consequenceState = this.State;
+            var consequenceResult = _result;
+
+            var alternative = VisitConditionalOperand(alternativeState, node.Alternative, isByRef);
+            Unsplit();
+            var alternativeResult = _result;
+
+            bool? getIsNullable(Result result) => (object)result.Type == null ? true : result.Type.IsNullable;
```

Comment:
If we're going to need this more generally, consider moving to `Result` type.

Reply:
I considered moving it to `Result` although I decided not to because it wasn't clear if other callers (when there are other callers) would won't the same behavior if `Type is null`. Will move it later when needed.